cogs/poll.py

```diff
@@ -1,15 +1,14 @@
 import os
 
-import discord
 from discord.ext import commands
 
 import utils
-from help.help import help, handle_error, help_category
+from cogs.help import help, handle_error, help_category
+from cogs.components.poll import Poll as ThePoll
```

Comment:
Zu #45 Hier liegt das Problem.  `Poll` wird hier importiert aber in dieser Datei gibt es durch den Wegfall des Suffixes `Cog` auch schon eine Klasse Poll. Da wird ein Verschieben in ein anderes Verzeichnis auch nicht helfen (ist ja schon in Verzeichnis `components`)